### ISSUE TITLE
App crashes since version 1.2.8 f-droid flavor

### ISSUE THREAD

**Issue by Menelmacar** [caydey/ffshare#62]
Since the upgrade to version 1.2.8, every time the app ends with an "error compressing media", regardless of the input.
A fresh install doesn't change that. Everything worked fine with the other versions.
(from f-droid, on a Pixel7, android 13)

**Comment by caydey** [comment]
Huge apologies, the issue is created by the new upgraded version of FFmpeg the app uses which I only noticed once publishing the release to F-Droid. I thought I had canceled the release but must've missed something.

Unfortunately the new release I've published fixing this will take around 5 days for F-Droid to update, and until then the latest version (v1.2.8) on F-Droid will be useless.

If you wish to use the app try downgrading it on F-Droid by selecting "Versions > 1.2.7" or by installing the APK directly from GitHub [FFShare_1.2.7_full.apk](https://github.com/caydey/ffshare/releases/download/v1.2.7/FFShare_1.2.7_full.apk)
